client/rpc-core/txpool/Cargo.toml

```diff
@@ -8,12 +8,12 @@ license = 'GPL-3.0-only'
 repository = 'https://github.com/PureStake/moonbeam/'
 
 [dependencies]
-ethereum = { version = "0.7.1", default-features = false, features = ["with-codec"] }
+ethereum = { version="0.7.1", default-features=false, features=["with-codec"] }
```

Comment:
Looks like a lot of this diff is just whitespace changes, that might cause a lot of headaches down the road

Reply:
Yes, and it doesn't seem to apply to all Cargo.toml, any reason for that ?

Reply:
Huh, I didn't do that on purpose. I wonder if my editor was trying to outsmart me.

Reply:
I think that was your editor probably, it looks like it touched lines that weren't otherwise updated. Anyway, I fixed it and force pushed (I'm now hoping that doesn't cause further problems!)